docs/stable/2-account.md

```diff
@@ -213,12 +220,193 @@ All messages sent are encrypted with the public key of the destination and signe
 
 -->
 
+## Public Key Compression
```

Comment:
this is serialization - it's not compression, as in zip files. when serializing, you have the option to do so in a compressed format which replaces one of the two coordinates with a flag - you don't decompress it either, you deserialize it

Reply:
Most libraries call this functionality "compress" and "decompress", or "from_compressed", "to_compressed", "from_decompressed" and "to_decompressed". The common parlance for this functionality is compress and decompress.

To be technically accurate I can add to the Terms Glossary that the terms "public key compression" and "public key decompression" are colloquial terms for the technical terms "public key serialisation" and "public key deserialisation".

The target reader and beneficiary of this document is much more likely to search for and understand the terms compress and decompress, so my opinion is that we keep the idiomatic phraseology in the main body of the document for this reason.

Reply:
On reflection I've decided to implement this differently, I've taken your suggestion on board and now use the "serialize" terminology, with only a minor sentence detailing the idiomatic version of the technically correct phrases.